pkg/api/v1/types/types.go

```diff
@@ -152,6 +153,7 @@ func (c *ConditionUpdate) MergeExisting(existing *rctypes.Condition) (*rctypes.C
 		Parameters:            existing.Parameters,
 		State:                 c.State,
 		Status:                c.Status,
+		Target:                existing.Target,
```

Comment:
I feel like if we're going to guard against arbitrary bad data we should add a validation that `existing.Target` and `c.Target` should be the same

Reply:
fixed in [b187837](https://github.com/metal-toolbox/conditionorc/pull/203/commits/b1878374bb35b3f03f0d26b74f7342fe652352c2)